### PR TITLE
A role for configuring /etc/hosts on the nodes

### PR DESCRIPTION
**Pull request by iNecas**
Updates from me trying to use this stuff

### CODE REVIEW

**Comment by iNecas** [comment]
Also, I've added a script to update the localhost /etc/hosts based on the ansible inventory

File: ansible.cfg [hunk 1]
```diff
@@ -1,2 +1,4 @@
 [defaults]
 callback_plugins = playbooks/callback_plugins/
+nocows = 1
```
Comment (ehelms) [inline ansible.cfg:3]: Can I has sheep?

**Comment by ehelms** [comment]
@iNecas this tested well for me, how would you like to handle including this? Do you want to squash and me merge it in? or want me to squash merge? The PR title could use an update to be more descriptive before I merge this into my branch.

**Comment by iNecas** [comment]
I've updated the title, squash&merge works for me

**Comment by ehelms** [comment]
@iNecas I had to rebase my branch on master which led to conflicts which has led to your PR now having conflicts.

**Comment by iNecas** [comment]
Updated

**Pull request merged**